bevy_interact_2d/src/lib.rs

```diff
@@ -215,8 +218,10 @@ fn setup_interaction_debug(
     let child = commands
       .spawn_bundle(GeometryBuilder::build_as(
         &bounding_mesh,
-        ShapeColors::new(Color::rgb_u8(red, green, blue)),
-        DrawMode::Stroke(StrokeOptions::default()),
+        DrawMode::Outlined {
+          fill_mode: FillMode::color(Color::rgb_u8(red, green, blue)),
+          outline_mode: StrokeMode::new(Color::BLACK, 10.0),
+        },
```

Comment:
N.b. this is probably different now. I'm not sure what the old default stroke options were and I couldn't see anything when running the example on the old version of this code.